examples/tutorial 1a - Using a Quantum Device to solve MIS.ipynb

```diff
@@ -26,17 +57,41 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "import networkx as nx\n",
     "\n",
-    "from mis.shared.types import MISInstance\n",
-    "from mis.pipeline.config import SolverConfig\n",
-    "from mis import MISSolver"
+    "# Create a new networkx graph instance to be populated with DIMACS data.\n",
+    "graph = nx.Graph()\n",
+    "\n",
+    "\n",
+    "with open(\"./datasets/a265032_1tc.32.txt\", \"r\") as f:\n",
```

Comment:
Should we provide a utility to import this format automatically?

Perhaps as a (simple) UnitaryHack issue?

Reply:
Could be. Together with extending the test suite to other DIMACS datasets ?

Reply:
#29 Could you please review @Yoric ?

Reply:
(done)